cmd/soroban-rpc/internal/test/cli_test.go

```diff
@@ -62,13 +66,108 @@ func TestCLIContractDeployAndInvoke(t *testing.T) {
 	require.Contains(t, output, `["Hello","world"]`)
 }
 
+func TestCLISimulateTransactionBumpAndRestoreFootprint(t *testing.T) {
```

Comment:
This test doesn't really bump or restore, could you pick a more appropriate name?

Reply:
Ah, great!